assets/sass/authorize-application/_googlesitekit-authorize-application.scss

```diff
@@ -16,10 +16,138 @@
  * limitations under the License.
  */
 
-.auth-app-card #approve {
-	background-color: $c-deep-blue;
+.authorize-application-php {
+	background-color: $c-authorize-application-background;
+
+	.wrap {
+		margin: 30px 0 0;
+
+		h1 {
+			// This is the Authorize Application screen title.
+			font-size: $fs-authorize-application-headline-sm;
+			line-height: $lh-authorize-application-headline-sm;
+			padding: 0;
+		}
+	}
+
+	.auth-app-card {
+		border: none;
+		margin: 37px 0 0;
+		padding: 0;
+	}
+
+	.title {
+		font-size: $fs-authorize-application-title-lg;
+		font-weight: $fw-medium;
+		line-height: $lh-authorize-application-title-lg;
+		margin: 19px 0;
```

Comment:
It seems that setting the margin is not needed. It doesn't make any difference since the margin already has 18px at the top and 18px at the bottom by default.
![Screenshot 2024-04-25 at 5 48 52 PM](https://github.com/google/site-kit-wp/assets/24408261/2e50d650-0671-46cb-bf12-ddfbb388bf72)

Reply:
Good point, thanks! I've removed this margin declaration.